keep/api/arq_worker.py

```diff
@@ -1,29 +1,69 @@
-# builtins
+import os
+import logging
 from typing import Optional
 
-# third-party
 from arq import Worker, create_pool, cron
 from arq.connections import RedisSettings
 from arq.worker import create_worker
 from pydantic.utils import import_string
 from starlette.datastructures import CommaSeparatedStrings
 
-# internals
+import keep.api.logging
+from keep.api.logging import CONFIG as logging_config
 from keep.api.core.config import config
 from keep.api.tasks.process_background_ai_task import process_background_ai_task
 from keep.api.tasks.healthcheck_task import healthcheck_task
 
-ARQ_BACKGROUND_FUNCTIONS: Optional[CommaSeparatedStrings] = config(
-    "ARQ_BACKGROUND_FUNCTIONS",
-    cast=CommaSeparatedStrings,
-    default=[
+keep.api.logging.setup_logging()
+logger = logging.getLogger(__name__)
+
+###
+# Set ARQ_TASK_POOL_TO_EXECUTE to "none", "all", "basic_processing" or "ai" 
+# to split the tasks between the workers.
+###
+
+ARQ_TASK_POOL_TO_EXECUTE_NONE = "none"  # Arq workers explicitly disabled for this service
+ARQ_TASK_POOL_TO_EXECUTE_ALL = "all"  # All arq workers enabled for this service
+ARQ_TASK_POOL_TO_EXECUTE_BASIC_PROCESSING = "basic_processing" # Everything except AI
+ARQ_TASK_POOL_TO_EXECUTE_AI = "ai" # Only AI
+
+REDIS = os.environ.get("REDIS", "false") == "true"
+ARQ_TASK_POOL_TO_EXECUTE = os.environ.get("ARQ_TASK_POOL_TO_EXECUTE", None)
```

Comment:
I gotta say this is pretty confusing. Let's stick with easier solution like `AUTH_MODE`:
`KEEP_ARQ_MODE=none`
`KEEP_ARQ_MODE=all`
`KEEP_ARQ_MODE=basic_processing`

Reply:
I'm not sure it's "ARQ_MODE" is representing what I'm trying to do here. The env is configuring "Which tasks are scheduled for this worker". Maybe "KEEP_ARQ_TASK_POOL"?

Reply:
@Matvey-Kuk there's already ARQ_BACKGROUND_FUNCTIONS for that, no?